### ISSUE TITLE
Custom "disk offerings"with "custom disk size" not working after upgrade from 4.16.1

### ISSUE THREAD

**Issue by Hudratronium** [apache/cloudstack#6678]
##### ISSUE TYPE

 * Bug Report

##### COMPONENT NAME

~~~
Management Server
~~~

##### CLOUDSTACK VERSION

~~~
4.17.0.1
~~~

##### OS / ENVIRONMENT

Ubuntu 20.04

##### SUMMARY

After upgarding to 4.17.0.1 from 4.16.1 my cutome disk offerings with "custom disk size" option aren't working anymore.
Errormessage:
<img width="301" alt="custome_offering" src="https://user-images.githubusercontent.com/79725167/187185203-9a0ebebc-e3d2-4735-8dfa-d65616780594.png">

No log entries in the management-server.log

##### STEPS TO REPRODUCE

- Create a disk offering with option "custome disk size" in  CS 4.17.0.1
- Use disk offering for creation of a new instance
- Enter value for disk size
- launch instance

~~~

~~~

##### EXPECTED RESULTS

~~~
Instance is generated and started
~~~

##### ACTUAL RESULTS

~~~
Error in GUI; see screenshot above
~~~

**Comment by shwstppr** [comment]
@Hudratronium this has been fixed with https://github.com/apache/cloudstack/pull/6586.
You may use API to deploy such VMs with 4.17.0.1 packages. You may also try latest nightly packages, https://download.cloudstack.org/testing/nightly/latest/

**Comment by Hudratronium** [comment]
Thanks for this info. 
I am just looking but do we have already a planned releasewindow for 4.17.1?
As this disables a "customer" faceing function, a remark on this in the releasenotes for 4.17.0 / 4.17.0.1 might be usefull...

**Comment by shwstppr** [comment]
@Hudratronium https://markmail.org/message/jjbxfv6tlx6s2tj2

**Comment by Hudratronium** [comment]
Thanks - for me this is solved looking forward to the release of 4.17.1
